src/provider/Provider.ts

```diff
@@ -573,23 +571,22 @@ export class Provider {
     }
   }
 
-  /** Get status for a specific jobId/documentId/owner.
-   * @param {string} jobId
-   * @param {number} index
-   * @param {string} providerUri
-   * @param {string} destination
-   * @param {Web3} web3
-   * @param {AbortSignal} signal abort signal
+  /** Get compute result url
+   * @param {string} providerUri The URI of the provider we want to query
+   * @param {Web3} web3 Web3 instance
+   * @param {string} consumerAddress The consumer ethereum address
+   * @param {string} jobId The ID of a compute job.
+   * @param {number} index Result index
+   * @param {AbortSignal} signal Abort signal
```

Comment:
no `AbortSignal` param on the actual method, and the return type is different from this jsdoc comment too. So most likely comment has to be updated

Reply:
missed that 👍  updated the method too many times 😃